README.md

```diff
@@ -11,13 +11,13 @@ First, add this to your `Cargo.toml`:
 
 ```toml
 [dependencies]
-tokio = { git = "https://github.com/tokio-rs/tokio" }
+tokio_proto = { git = "https://github.com/tokio-rs/tokio_proto" }
```

Comment:
I think the line needs to read:
`tokio-proto = { git = "https://github.com/tokio-rs/tokio-proto" }`